gnomad_qc/v3/variant_qc/evaluation.py

```diff
@@ -0,0 +1,349 @@
+
+            ht = ht.annotate(score=metric_ht[ht.key].score,)
+
+            ht = compute_binned_truth_sample_concordance(ht, metric_ht, args.n_bins,)
+            ht = ht.annotate_globals(
+                filter_low_conf_regions=args.filter_low_conf_regions
+            )
+            ht.write(
+                get_binned_concordance(model_id, truth_sample).path,
+                overwrite=args.overwrite,
+            )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--slack_channel", help="Slack channel to post results and notifications to."
+    )
+    parser.add_argument(
+        "--overwrite",
+        help="Overwrite all data from this subset (default: False)",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--run_hash",
+        help="Run hash. Created by --train_rf and only needed for --apply_rf without running --train_rf",
+        required=False,
+    )
+
+    parser.add_argument(
+        "--vqsr",
+        help="When set, creates the VQSR rank file instead of an RF rank file.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--vqsr_type",
+        help="What type of VQSR was run: allele-specific, or allele-specific with transmitted singletons",
+        type=str,
+        choices=["classic", "alleleSpecific", "alleleSpecificTrans"],
+        default="alleleSpecificTrans",
+    )
+    parser.add_argument(
+        "--create_quantile_bin_ht",
+        help="When set, creates file annotated with quantile bin based on vqsr/RF run hash score.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--run_sanity_checks",
+        help="When set, runs ranking sanity checks.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--create_aggregated_bin_ht",
+        help="When set, creates a file with aggregate counts of variants based on quantile bins.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--n_bins",
+        help="Number of bins for the binned file (default: 100)",
+        default=100,
+        type=int,
+    )
+    parser.add_argument(
+        "--extract_truth_samples",
+        help="Extract truth samples from matrix table",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--n_partitions",
+        help="Desired number of partitions for output Table/MatrixTable",
+        default=5000,
+        type=int,
+    )
+    parser.add_argument(
+        "--merge_with_truth_data",
+        help="Computes a table for each truth sample comparing the truth sample in the callset vs the truth.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--bin_truth_sample_concordance",
+        help="Merges individual concordance results with specified metric binned files.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--filter_low_conf_regions",
```

Comment:
Good point. I added because when I was testing the code on v3 using VQSR my plots looked very different from yours. After testing I realized it was because I had added in the filter to low conf regions to the code so I added an option for it.